17437 Activity comment and details as formattable

hschink opened the pull request:
https://community.openproject.org/work_packages/17437

Commit:
Test that activity comment is a formattable

Commit:
Clean up activity endpoint specs

Commit:
Make activity comment a formattable

Commit:
Adapt frontend to activity comment format

Commit:
Test that activity details is a list of formattables

Commit:
Make activity details a list of formattables

Commit:
Adapt frontend activity details structure

TeatroIO commented:
I've prepared a stage. <a href='http://fix-17437-activity-comment-and-details-as-formattable.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

Commit:
Test that activity comment and details are correctly displayed

Commit:
Trust html received from the back-end

Commit:
Adapt spec to formattable activity comment

Commit:
Adapt at leat a few activities to the formattable layout

These are by far not all entries affected that need to be adapted to activity comment/details supporting formattable.

Commit:
Test that the last 3 activities are displayed on the overview tab
